code/datums/status_effects/buffs.dm

```diff
@@ -13,6 +13,7 @@
 /datum/status_effect/shadow_mend/on_apply()
 	owner.visible_message("<span class='notice'>Violet light wraps around [owner]'s body!</span>", "<span class='notice'>Violet light wraps around your body!</span>")
 	playsound(owner, 'sound/magic/Teleport_app.ogg', 50, 1)
+	return ..()
```

Comment:
@RemieRichards has told me different, that apparently the `return` keyword is somehow shitter than just the functioning ending.